index.js

```diff
@@ -0,0 +1,18 @@
+const delayedPromiseChain = async (promiseFunctions, delay) =>{
```

Comment:
please implement this solution using **_.then syntax_** instead of async await, you can discuss it with me. there are some **_restrictions_** for this problem which I will let u know.